xbmc/interfaces/json-rpc/ProfilesOperations.cpp

```diff
@@ -91,7 +91,7 @@ JSONRPC_STATUS CProfilesOperations::LoadProfile(const std::string &method, ITran
   std::string profilename = parameterObject["profile"].asString();
   int index = profileManager->GetProfileIndex(profilename);
 
-  if (index < 0)
+  if (index < MASTER_PROFILE_ID)
```

Comment:
Maybe `if (index <= INVALID_PROFILE_ID)`? That seems to fit the logic above.

Reply:
Yep, will change that.